xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -510,22 +510,18 @@ bool CDRMUtils::OpenDrm(bool needConnector)
 
     for (auto module : modules)
     {
-      m_fd = drmOpen(module, device.c_str());
+      m_fd.attach(drmOpen(module, device.c_str()));
       if (m_fd >= 0)
       {
         if(!GetResources())
         {
-          drmClose(m_fd);
```

Comment:
Is `drmClose` equivalent to `close`?

Reply:
They can be used interchangeably (drmClose does do some extra things but I'm not worried about it). I only use drmOpen because it allows us to whitelist modules that are known to work.

Reply:
I suggest adding a comment to that effect